sources/academy/tutorials/node_js/handle_blocked_requests_puppeteer.md

```diff
@@ -14,20 +14,21 @@ One of the main defense mechanisms websites use to ensure they are not scraped b
 Setting proxy rotation in [BasicCrawler](https://crawlee.dev/api/basic-crawler/class/BasicCrawler) is pretty simple. When using plain HTTP requests (like with the popular '[request-promise](https://www.npmjs.com/package/request-promise)' npm package), a fresh proxy is set up on each request.
 
 ```js
-const Apify = require('apify')
-const requestPromise = require('request-promise')
-const PROXY_PASSWORD = process.env.APIFY_PROXY_PASSWORD
-const proxyUrl = `http://auto:${PROXY_PASSWORD}@proxy.apify.com`
+const Apify = require('apify');
+const requestPromise = require('request-promise');
+
+const PROXY_PASSWORD = process.env.APIFY_PROXY_PASSWORD;
+const proxyUrl = `http://auto:${PROXY_PASSWORD}@proxy.apify.com`;
 
 const crawler = new Apify.BasicCrawler({
-    requestList: someInitializedRequestList,
-    handleRequestFunction: async ({request}) => {
-        const response = await requestPromise ({
-            url: request.url,
-            proxy: proxyUrl
-        })
-   }
-})
+    requestList: someInitializedRequestList,
+    handleRequestFunction: async ({ request }) => {
```

Comment:
there will be probably more examples like this based on v2, this is nowadays called `requestHandler`, and the import should be from `crawlee`, not `apify`. those are probably worth fixing in a separate PR

Reply:
Yeah, those are some old not-important articles someone copied without much thought, I will have another pass on them in the future